apps/web/app/s/[surveyId]/page.tsx

```diff
@@ -1,5 +1,32 @@
-import SurveyPage from "./SurveyPage";
+export const revalidate = REVALIDATION_INTERVAL;
 
-export default function LinkSurveyPage({ params }) {
-  return <SurveyPage surveyId={params.surveyId} />;
+import { REVALIDATION_INTERVAL } from "@formbricks/lib/constants";
+import LinkSurvey from "@/app/s/[surveyId]/LinkSurvey";
+import LegalFooter from "@/components/shared/LegalFooter";
```

Comment:
Can you please move the `LegalFooter` component to the same folder as it's only used here? 😊